pkg/crcontrollers/operator/common/annotation.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright © 2021 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	k8sClient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/dell/csi-baremetal/pkg/base/featureconfig"
+)
+
+// GetNodeIDByName return special id for k8sNode with nodeName
+// depends on NodeIdFromAnnotation and ExternalNodeAnnotation features
+func GetNodeIDByName(client k8sClient.Client, nodeName string, annotationKey string, featureChecker featureconfig.FeatureChecker) (string, error) {
```

Comment:
I think, it's better to use pointer receivers for client

Reply:
k8sClient.Client is interface and doesn't need pointer, wrapped KubeClient isn't needed here too. Or do I misunderstand anything?